style.css

```diff
@@ -4,4 +4,171 @@
   height: 2766px;
   width: 1440px;
 }
-\n
\ No newline at end of file
+
+.header {
+  background-image: url(images/nav/pic-bgr.jpg);
+  height: 839px;
+  width: 1440px;
+}
+
+.nav {
+  display: flex;
+  margin-left: 585px;
+  width: 1440px;
+}
+
+.nav__link { 
+  color: #fff;
+  font-size: 18px;
+  padding-top: 70px;
+  text-decoration: none;
+}
+
+.nav__link:first-child {
+  height: 21px;
+  padding-right: 60px;
+  width: 152px;
+}
+
+.nav__link:nth-child(2) {
+  padding-right: 60px;
+}
+
+.nav__link:nth-child(3) {
+  padding-right: 60px;
+}
+
+.nav__link:nth-child(4) {
+  padding-right: 130px;
+}
+
+.nav__link:hover {
+  color: #fff;
+  cursor: pointer;
+  text-decoration: underline;
+}
+
+.introduce {
+  color: #e1e6e9;
+  display: flex;
+  flex-direction: column;
+  font-weight: 900;
+  margin-bottom: 570px;
+  margin-left: 959px;
+  margin-right: 130px;
+  margin-top: 131px;
+}
+
+.introduce__title {
+  font-size: 40px;
+  margin: 0;
+  padding-bottom: 11px;
+}
+
+.introduce__paragraph {
+  font-size: 20px;
+  margin: 0;
+  padding-bottom: 39px;
+  text-align: right;
+}
+
+.button {
+  font-size: 20px;
+  font-weight: 900;
+  height: 55px;
+  text-align: center;
+  width: 200px;
+}
+
+.button_mediumvioletred {
+  align-self: flex-end;
+  background-color: #e0036f;
+  color: #fff;
+}
+
+.tickets {
+  align-items: center;
+  background-color: #e3eafe;
+  display: flex;
+  flex-direction: column;
+  height: 397px;
+  width: 1440px;
+}
+
+.title {
+  color: #585858;
+  font-weight: 500;
+  font-size: 30px;
+}
+
+.tickets__title {
+  height: 70px;
+  margin: 0;
+  padding-bottom: 40px;
+  padding-top: 55px;
+  text-align: center;
+  width: 621px;
+}
+
+.tickets__paragraph {
```

Comment:
Codacy found an issue: [Expected "#263B83" to be "#263b83" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/IFW.2/pullRequest?prid=6289680)